### PR TITLE
DM-42741: Make focalPlaneGeometryPlot look a bit better

### PR DESCRIPTION
**Pull request by sr525**

### CODE REVIEW

File: python/lsst/analysis/tools/actions/plot/focalPlanePlot.py [hunk 1]
```diff
@@ -343,6 +355,10 @@ def makePlot(
         fig : `matplotlib.figure.Figure`
             The resulting figure.
         """
+
+        cmap = mkColormap(["midnightBlue", "lightcyan", "darkgreen"])
```
Comment (plazas) [inline python/lsst/analysis/tools/actions/plot/focalPlanePlot.py:359]: Why are there 3 options here, and could we add "tableau-colorblind10"?

File: python/lsst/analysis/tools/actions/plot/focalPlanePlot.py [hunk 2]
```diff
@@ -487,8 +503,23 @@ def makePlot(
...
+            vmin = statMed - 4.0 * statMad
+        if self.vmax:
+            vmax = self.vmax
+        else:
+            vmax = statMed + 4.0 * statMad
+
+        valuesPlot = np.clip(values, vmin, vmax)
+
+        patchCollection = PatchCollection(
+            patches, edgecolor="white", cmap=cmap, linewidth=0.5, linestyle=(0, (0.5, 3))
```
Comment (plazas) [inline python/lsst/analysis/tools/actions/plot/focalPlanePlot.py:520]: would it be too much to make these options (e.g., edgecolor, linewidth...) configurable too, or is it not worth it?

**Review by plazas — APPROVED** [review]
LGTM! Just a few questions for clarification.

File: python/lsst/analysis/tools/actions/plot/focalPlanePlot.py (hunk 2 above)
Comment (sr525) [reply python/lsst/analysis/tools/actions/plot/focalPlanePlot.py:520]: I would prefer not to make all of these things configurable. I would like the plots produced to be uniform and not clutter up the code with too many config options.

**Pull request merged**